src/content/platform-integration/web/faq.md

```diff
@@ -120,18 +120,109 @@ See [Preparing a web app for release][].
 
 Not currently.
 
+### Why doesn't my app update immediately after it's deployed?
+
+You may need to configure the Cache-Control header returned by your web server. 
+For example, if this header is set to 3600, then the browser
+and CDN will cache the asset for 1 hour, and your users may see an out-of-date
+version of your app up to 1 hour after you deploy a new version. For
+more information about caching on the web,
+see [this web.dev article][http-cache] on HTTP caching.
+
+It is a good idea to be aware of this behavior to avoid breaking users.
```

Comment:
```suggestion
It is a good idea to be aware of this behavior to avoid an undesirable user experience.
```

?? They aren't actually breaking users.  :)